docker/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Missing license header, pls.

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/3a5d1c5f5d74cc9b34444041157ce0f03f3ef7f0

Reply:
(Actually it seems it was 04ecf9c897835733a9715527617f84f1a790343c)